Git-Discard-Unstaged-Changes.md

```diff
@@ -0,0 +1,11 @@
+# Discard unstaged changes
+
+When using Git, it is common to make changes that you want to remove entirely before the staging phase. For example, after working on a few files, you realize that you want to revert the changes made to one specific file. To discard the changes before staging and committing, use the `$ git checkout` command. 
+
+#### To unstage one file :       
+`$ git checkout <path-to-file>`  
+
```

Comment:
Thanks for being patient. Do you mean putting three backticks instead of one on line 6? Is there a difference between one and three backticks for a single line of code? I read that there was no difference.